File: DropZone/DropZone.Web/src/models/dropzones/dropzone.model.ts
import { AircraftModel } from "../aircrafts/aircraft.model";
import { UserModel } from "../users/user.model";

export class DropZoneModel {
	public Id: number;
	public Title: string;
	public Country: string;
	public RunwayType: string;
	public RunwayLength: number;
	public Square: number;
	public Latitude: number;
	public Longitude: number;
	public RegistrationDate: Date;
	public ModifiedAt: Date;
	
	public Aircrafts: AircraftModel[];
	public Users: UserModel[];
};

File: DropZone/DropZone.Core/Models/DropZone/DropZoneModel.cs
using System;
using System.Collections.Generic;

namespace DropZone.Core.Models
{
    public class DropZoneModel
    {
        public long Id { get; set; }

        public string Title { get; set; }

        public string Country { get; set; }

        public string RunwayType { get; set; }

        public double RunwayLength { get; set; }

        public double Square { get; set; }

        public double Latitude { get; set; }

        public double Longitude { get; set; }

        public DateTime RegistrationDate { get; set; }

        public DateTime ModifiedAt { get; set; }


        public List<AircraftModel> Aircrafts { get; set; }

        public List<UserModel> Users { get; set; }
    }
}
